Boards/heltec-wifi-lora-32-v3/Source/devices/Constants.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "driver/gpio.h"
+#include "driver/i2c.h"
+#include "driver/spi_common.h"
```

Comment:
Minor: Use '< >' for non-local includes.
'" "' prioritises local files, but these files are not local.